resources/assets/coffee/react/beatmap-discussions/nominations.coffee

```diff
@@ -233,7 +233,7 @@ class BeatmapDiscussions.Nominations extends React.PureComponent
                 className: "#{bn}__note #{bn}__note--nominators"
                 dangerouslySetInnerHTML:
                   __html: osu.trans 'beatmaps.nominations.nominated_by',
-                    users: nominators.map (user) ->
+                    users: nominators.reverse().map (user) ->
```

Comment:
I wonder if it's better to unshift instead of push when generating the array. A bit clearer about the order or something.